src/spid-sdk.js

```diff
@@ -27,6 +28,15 @@ function init(opts, callback) {
     }
 }
 
+function tryVarnishCookie(session) {
```

Comment:
Hm.. It could be in the cookie module, then we'd probably just expose tryVarnishCookie in public interface and make setVarnishCookie private?